src/getPortalSdk.ts

```diff
@@ -2,6 +2,34 @@ import { GraphQLClient } from 'graphql-request';
 import { getConnectionCookie } from './cookies';
 import { getSdk, ShareInput } from './sdk';
 
+export enum trackingEventList {
```

Comment:
nit: title case for enum `TrackingEventList`

Reply:
cb1fd351549d60675574453f5bc999c7ea07ac42